in_progress/roles.md

```diff
@@ -0,0 +1,91 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the `scverse` project, to clarify how decisions are made and how the various elements of our community interact.
+
+As a whole, we are a consensus-based community project.
+However, any subproject within the scverse can have their own governance system as long as they do not conflict with the rules set out here.
+Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
+
+## Contributors/ Project developers
+
+A community member can become a contributor by interacting directly with the project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification via a GitHub pull request to any https://github.com/scverse repository;
+- reporting a GitHub issue to any https://github.com/scverse repository;
+- discussing examples or usage issues on any of our discussion platforms
+
+among other possibilities.
+Any community member can become a contributor, and all are encouraged to do so.
+By contributing to the project, community members can directly help to shape its future.
+
+All community members are required to adhere by our code of conduct (**TODO: add link**).
+Potential contributors are encouraged to read the Contributing Guide (**TODO: add link**).
+
+A community member becomes a contributor when the following criteria are met:
+
+- At least two core developers from separate employers agree membership is a good idea.
+- The new member has supported the project several times, either through code or otherwise, and has demonstrated a willingness to help solve other people's problems, not just their own.
```

Comment:
I agree with the point but maybe the wording not optimal? I would remove "not just their own" and add "and activily engaging in community projects." Basicalluy people that are active on github but don't necessarily write code.

Reply:
more generally, for me contributors are people that are not listed on website, but that have the badge on github (and potentially the linkedin badge as well) and are active contributors of discussion in various forms other than actually writing code

Reply:
```suggestion
- The new member has supported the project several times, either through code or otherwise
```

I would be fine with this even. I feel like supporting the project goes with solving problems.

E.g. asking many questions isn't really supporting, but pointing other users to the right solutions is

Reply:
> E.g. asking many questions isn't really supporting, but pointing other users to the right solutions is

true, agree